### ISSUE TITLE
What about printing support?

### ISSUE THREAD

**Issue by PeterWone** [craigcecil/vscode-design-in-browser-pack#1]
There are two extensions that can provide this. Both are cross platform by exploiting the local browser's ability to print. 

* VS Code Printing Free
* PrintCode

PrintCode has a number of defects and the project appears to be abandoned. I wrote VS Code Printing Free to address all the outstanding issues and also my own wishlist.

**Comment by craigcecil** [comment]
Peter--thanks for the head's up about those two extensions. I'll check them out and let you know if they add value to the extension pack.

**Comment by craigcecil** [comment]
Checked out both extensions. Both look good. I did notice that PrintCode is not abandoned--the repo maintainer is currently answering issues and has pushed a couple minor updates in the past two months. At this time I'm leaning towards not adding printing support to the extension pack, since I want to keep the pack focused on design in the browser functionality exclusively. Users can always add printing support on their own. Thanks again for alerting my to these two extensions.
